src/server/test/unit/LineReader.test.ts

```diff
@@ -0,0 +1,98 @@
+    var TextDocument = { getText: function (range?: Range) {return '<div class="row sectionBlockLayout" style="display: flex; flex-wrap: wrap; padding: 8px; margin: 0px; text-align: center; min-height: auto;"></div>\n<div>en</div>\n<div>entity</div>'},  range:1, _content:'<div class="row sectionBlockLayout" style="display: flex; flex-wrap: wrap; padding: 8px; margin: 0px; text-align: center; min-height: auto;"></div>\n en',
+    offsetAt:1, } as unknown as TextDocument;
+   const manifest= getEditedLineContent(1, TextDocument);
+    expect(manifest).to.be.not.undefined;
+    expect(manifest).eq("<div>en</div>");
+     
+   });
+   it('LineReader- test getEditedLineContent for 5 line content (edited 4th line)', () => {
+    var TextDocument = { getText: function (range?: Range) {return '<div class="row sectionBlockLayout" style="display: flex; flex-wrap: wrap; padding: 8px; margin: 0px; text-align: center; min-height: auto;">testLin1</div>\n<div>testLin2</div>\n<div>testLin3</div>\n<div>testLin4</div>\n<div>testLin5</div>'},  range:1, _content:'<div class="row sectionBlockLayout" style="display: flex; flex-wrap: wrap; padding: 8px; margin: 0px; text-align: center; min-height: auto;"></div>\n en',
+    offsetAt:1, } as unknown as TextDocument;
+   const manifest= getEditedLineContent(3, TextDocument);
+    expect(manifest).to.be.not.undefined;
+    expect(manifest).eq("<div>testLin4</div>");
+     
+   });
+
+   it('LineReader- test getEditedLineContent for 5 line content (edited 2nd line)', () => {
+    var TextDocument = { getText: function (range?: Range) {return '<div class="row sectionBlockLayout" style="display: flex; flex-wrap: wrap; padding: 8px; margin: 0px; text-align: center; min-height: auto;">testLin1</div>\n<div>testLin2</div>\n<div>testLin3</div>\n<div>testLin4</div>\n<div>testLin5</div>'},  range:1, _content:'<div class="row sectionBlockLayout" style="display: flex; flex-wrap: wrap; padding: 8px; margin: 0px; text-align: center; min-height: auto;"></div>\n en',
+    offsetAt:1, } as unknown as TextDocument;
+   const manifest= getEditedLineContent(1, TextDocument);
+    expect(manifest).to.be.not.undefined;
+    expect(manifest).eq("<div>testLin2</div>");
+     
+   });
+
+
+   it('LineReader- test getEditedLineContent for 4 line content (edited 4th line)', () => {
+    var TextDocument = { getText: function (range?: Range) {return '<div class="row sectionBlockLayout" style="display: flex; flex-wrap: wrap; padding: 8px; margin: 0px; text-align: center; min-height: auto;">testLin1</div>\n<div>testLin2</div>\n<div>testLin3</div>\n<div>testLin4</div>'},  range:1, _content:'<div class="row sectionBlockLayout" style="display: flex; flex-wrap: wrap; padding: 8px; margin: 0px; text-align: center; min-height: auto;"></div>\n en',
+    offsetAt:1, } as unknown as TextDocument;
+   const manifest= getEditedLineContent(3, TextDocument);
+    expect(manifest).to.be.not.undefined;
+    expect(manifest).eq("<div>testLin4</div>");
+     
```

Comment:
style/readability nit: please clean up the extraneous empty lines, e.g. after the last expect(); or the more than one empty line between it() blocks

Reply:
done

Reply:
@davidjenni *please review/approve now